packages/swc/package.json

```diff
@@ -0,0 +1,73 @@
+{
+  "name": "@rollup/plugin-swc",
+  "version": "0.0.0",
+  "publishConfig": {
+    "access": "public"
+  },
+  "description": "Transpile JavaScript/TypeScript code with swc.",
+  "license": "MIT",
+  "repository": {
+    "url": "rollup/plugins",
+    "directory": "packages/swc"
+  },
+  "author": "Peter Placzek <peter.placzek1996@gmail.com>",
+  "homepage": "https://github.com/rollup/plugins/tree/master/packages/swc#readme",
+  "bugs": "https://github.com/rollup/plugins/issues",
+  "main": "dist/cjs/index.js",
+  "module": "dist/es/index.js",
+  "exports": {
+    "types": "./types/index.d.ts",
+    "import": "./dist/es/index.js",
+    "default": "./dist/cjs/index.js"
+  },
+  "engines": {
+    "node": ">=10.0.0"
```

Comment:
Should this be ">=14.0.0" to be the same as the other plugins and since that's what's tested?

Reply:
swc and rollup v2 (https://www.npmjs.com/package/rollup/v/2.77.4-1?activeTab=explore) require a version of `>=10.0.0`. If someone installs rollup v3 instead the new engine requirement will be provided by the rollup dep ;)

Reply:
Seems odd to be inconsistent with the other plugins though: ae4e9b3718c6bb9714443dac9cfa1dca6210480b

Reply:
this min engine version corresponds to the min engine version of [slash](https://www.npmjs.com/package/slash?activeTab=explore) and slash is dependency, so its usage is mandatory. Andjust because other plugins have a different value, this one does not automatically have the same. I have already made considerations or research before I have adapted anything :v: :relaxed:

Reply:
We don't support Node versions out of LTS. Minimum rollup version should be v3 and Node version 14.

Reply:
@tada5hi this is still pending

Reply:
sorry, I had misinterpreted that. :see_no_evil:

Reply:
but is also done but is also solved by the latest commit

Reply:
The `rollup` version under `peerDependencies` wasn't updated to remove v2.